test/extended/router/unprivileged.go

```diff
@@ -19,7 +19,7 @@ import (
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
-var _ = g.Describe("[sig-network][Feature:Router][apigroup:route.openshift.io][apigroup:config.openshift.io][apigroup:template.openshift.io]", func() {
+var _ = g.Describe("[sig-network][Feature:Router][apigroup:route.openshift.io][apigroup:config.openshift.io]", func() {
```

Comment:
I think "[apigroup:template.openshift.io]" should still be here in this commit because the `BeforeEach` block uses the `router-common.yaml` fixture on line 52.  Again, not really an issue if a subsequent commit removes the use of `router-common.yaml`.